app/questionPage.html

```diff
@@ -73,16 +53,40 @@ <h3 id="timer"></h3>
                 <div class="question-container col-md-8">
                     <div id="questions"></div>
                     <div id="question-nav" class="question-nav"></div>
-                    <button type="button" id="finish" class="btn btn-success finish">Finish</button>
                 </div>
-                <div class="col-md-2">
+                <div class="col-md-2 overview_column">
                     <button type="button" class="btn btn-info next">Next</button>
+                    <button type="button" class="btn btn-info overview">Overview</button>
                 </div>
                 <div class="col-md-2">
                 </div>
             </div>
         </div>
     </div>
+    <div id="overview_page">
+        <h1>Test Overview</h1>
+        <table class="overview_page_table">
+            <thead>
+                <tr>
+                    <th>Question Id</th>
+                    <th>Question</th>
+                    <th>Answered</th>
+                    <th>Flagged</th>
+                </tr>
+            </thead>
+            <tbody class="overview_table_body"></tbody>
+        </table>
+        <footer class="overview_footer">
+            <div class="row">
+                <div class="col-xs-9">
+                    <p>Clicking this button submits the test once and for all there is no turning back!</p>
+                </div>
+                <div class="col-xs-3">
+                    <button type="button" id="finish" class="btn btn-danger finish">Finish</button>
```

Comment:
This button does nothing. I assume you will fix it when it comes to the Great Mergening(tm)

Reply:
Shows result page now, solved in >> f83be4e